src/jit/block.h

```diff
@@ -925,14 +925,12 @@ typedef unsigned weight_t;             // Type used to hold block and edge weigh
 
     bool endsWithJmpMethod(Compiler *comp);
 
-#if FEATURE_FASTTAILCALL
```

Comment:
It is not correct to use these routines on targets where FEATURE_FASTTAILCALL is not enabled, as they are checking for invariants that may not reflect the reality.